try:
    print(x)
except:
    print("Ha ocurrido algun error")
